Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -16,23 +16,23 @@ Alpha signals that we are ready for early feedback, that this is not the final v
 
-Reducing an SCI score is only possible through the elimination of emissions through changes to a software system to use less physical hardware, less energy, or consume lower-carbon energy sources. Neutralization or avoidance offsets do not reduce an SCI score ([see exclusions section](#exclusions)). This makes the SCI an ideal strategy organizations can adopt to meet climate targets focussed on eliminating emissions, such as a [Net-zero target](https://sciencebasedtargets.org/net-zero).
+Reducing an SCI score is only possible through the elimination of emissions. That can be achieved by changes to a software system to use less physical hardware, less energy, or consume lower-carbon energy sources. Neutralization or avoidance offsets do not reduce an SCI score ([see exclusions section](#exclusions)). This makes the SCI an ideal strategy that organizations can adopt to meet climate targets focussed on eliminating emissions, such as a [Net-zero target](https://sciencebasedtargets.org/net-zero).
 
 The SCI is for everyone. It is possible to calculate an SCI score for any software application, from a large distributed cloud system to a small monolithic open source library, any on-premise application or even a serverless function. The product or service may be running in any environment, whether a personal computer, private data center or a hyperscale cloud.
 
-As software practitioners, we all have a significant role to play in collectively reducing the SCI Score during the design, development and delivery of software applications. The following list provides some strategies that can be used to reduce the SCI score across software roles. 
+As software engineers, we all have a significant role to play in collectively reducing the SCI Score during the design, development and delivery of software applications. The following list provides some strategies that can be used to reduce the SCI score across software roles. 
 - For a software programmer, this would imply writing energy efficient code.
-- For a AL/ML developer it would imply model optimization, using pre-trained models or leveraging optimized hardwares for training
+- For a AI/ML developer it would imply model optimization, using pre-trained models or leveraging optimized hardware for training.
```

Comment:
```suggestion
- For an AI/ML developer it would imply model optimization, using pre-trained models or leveraging optimized hardware for training.
```